alerts/notifications.py

```diff
@@ -0,0 +1,36 @@
+from django.core.mail import send_mail
+from users.models import User
+from series.models import APIShow
+
+
+class NotificationBackend:
+    """Backend for generic notifications."""
+
+    def notify(self, user: User, tv_show: APIShow):
+        pass
+
+
+class EmailNotifier(NotificationBackend):
+    """Notifications with emails."""
+    _subject = 'A new episode arrives today !'
+    _basic_content = 'Today, a new episode of : '
+    _mail_from = 'new_series@uptv.com'
+
+    def notify(self, user: User, tv_show: APIShow):
+        send_mail(
+            self._subject,
+            self.get_full_content(user, tv_show),
+            self._mail_from,
+            [user.email],
+            fail_silently=False,
+        )
+
+    def get_full_content(self, user: User, tv_show: APIShow):
```

Comment:
Consider turning these into private methods (`_get_full_content()`) as they're not used outside of the class?

Reply:
Also looks like the `user` argument is not used here.